sdk/eventgrid/azure-messaging-eventgrid/README.md

```diff
@@ -152,6 +157,27 @@ EventGridPublisherAsyncClient<CloudEvent> cloudEventAsyncClient = new EventGridP
     .buildCloudEventPublisherAsyncClient();
 ```
 
+##### Using endpoint and Azure Active Directory (AAD) token credential to create the client
```

Comment:
It's good to update the changelog in the same PR that introduces new features so that we don't forget to include them before release.